ups/eupspkg.cfg.sh

```diff
@@ -13,7 +13,10 @@ config()
     # TokuDB requires third-party software jemalloci (see http://www.canonware.com/jemalloc/)
     ARGS+=('-DPLUGIN_TOKUDB=NO')
 
-    # Always use bundled PCRE
+    # Due to cmake library discovery being overly energetic (searching every
+    # lib directory relative to each entry in $PATH) we use the bundled ZLIB and
+    # PCRE libraries to avoid link confusion downstream
+    ARGS+=('-DWITH_ZLIB=bundled')
```

Comment:
The commit message isn't really clear about this ZLIB change.

Reply:
Oh.  It was part of "Streamline eupspkg.cfg to be more similar with mariadbclient version", but it certainly wouldn't hurt to be more explicit; will fix.

Still losing on OSX, BTW :-/  For some reason, -DWITH_SSL=off doesn't seem to bypass the failing GnuTLS check, though it seems by my first reading of the cmake file that it should.  Installing GnuTLS via homebrew "fixes" it, but I don't think we want to add another prereq if we can avoid it.  I'll continue to fiddle with it...